src/main/java/utils/ResponseWriter.java

```diff
@@ -0,0 +1,82 @@
+package utils;
+
+import http.Header;
+import http.HttpResponse;
+import http.HttpStatus;
+import http.HttpVersion;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.DataOutputStream;
+import java.io.IOException;
+
+
+public class ResponseWriter {
+
+    private static final Logger logger = LoggerFactory.getLogger(ResponseWriter.class);
+
+    private static final String CRLF = "\r\n";
+    private final HttpResponse httpResponse;
```

Comment:
```suggestion
    private static final String CRLF = "\r\n";
    
    private final HttpResponse httpResponse;
```
상수와 상태값은 구분 지어주면 코드의 가독성이 높아 집니다 :)

Reply:
감사합니다ㅎㅎ 피드백 반영했습니다~! 
[5418a0c](https://github.com/next-step/jwp-was-kakao/pull/6/commits/5418a0cf09e677ad1910fc037f319a5b8f20c28f)